### DISCUSSION TITLE
The difference of Peak/Cell matrix and TSS usage

### DISCUSSION THREAD

**Discussion by beginner984** [GreenleafLab/ArchR#760] [Questions / Documentation]
Hello

I need your help please to understand this

Cell-ranger atac pipeline returns two files **_filtered_peak_bc_matrix.h5_** and **_filtered_tf_bc_matrix.h5_**

Extract which from fragment file?

Signac software uses **_Peak/Cell matrix_** by getting **_filtered_peak_bc_matrix.h5_**

What if in your software I use **_Peak/Cell matrix_** instead? 

My question please is, does your sofware tries `TSS `enrichment and Signac package uses something equivalent to gene expression in transcriptome?

Thank you so much for any help

**Comment by rcorces** [comment]
I would encourage you to read the manual, publication, and supplementary material from the publication in full before posting questions.

> Cell-ranger atac pipeline returns two files filtered_peak_bc_matrix.h5 and filtered_tf_bc_matrix.h5

ArchR does not use either of these files.

> What if in your software I use Peak/Cell matrix instead?

This is not possible in ArchR.
